modules/machine-pools-hcp.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="machine-pools-hcp_{context}"]
+= Machine pools in {hcp-title} clusters
+
+In {hcp-title} clusters, the {hcp} spans across three availability zones (AZ) in the installed cloud region. Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. Each of these AZs can only have one machine pool. 
+
+Additionally, machine pools in {hcp-title} clusters each upgrade independently and the versions of the machine pools must remain within two minor (Y-stream) versions of the control plane. While machine pools created with your cluster use the same version, since machine pools upgrade independently, the machine pools must remain within 2 y-stream versions of the hsted control plane.
+
+The following images depict how machine pools work within ROSA and {hcp-title} clusters:
+
+image::hcp-rosa-machine-pools.png[Machine pools on ROSA classic and ROSA with HCP clusters]
+
+[NOTE]
+====
+Unlike ROSA classic machine pools, which match the control plane's Y-stream versions, machine pools in {hcp-title} clusters each upgrade independently and the versions of the machine pools must remain within two minor (Y-stream) versions of the control plane.
```

Comment:
```suggestion
Machine pools in {hcp-title} clusters each upgrade independently and the machine pool versions must remain within two minor (Y-stream) versions of the control plane.
```
I'm not sure if the comparison to ROSA Classic is necessary. If it a customer switching from Classic, it could be useful but otherwise, it just bogs down the text.